### DISCUSSION TITLE
Query for fetching value from a particular index in the array type field

### DISCUSSION THREAD

**Discussion by Palak-K9** [neo4j-graphql/neo4j-graphql-java#320] [Q&A]
I am using the below schema in which the field **link** has String Array as type.

```
type Customer{
 Id: Int!
 name: String!
 link: [String!]!
}
```
I want to fetch only a **particular index value** in the link field. For Eg. I want to fetch the value at 1st index in the link array.
Is there something like this to achieve the same? The below query isn't working.

```
{
 customers{
    link[1]
 }
```

**Comment by Andy2003** [comment]
The augmented API does not support this use case, but you can try to achieve this with a custom `@cypher` field

**Reply by Palak-K9** [reply]
Okay, is there any plan to implement it in future?
